src/map/clif.c

```diff
@@ -14060,20 +14058,20 @@ static void clif_PartyBookingInsertNotify(struct map_session_data *sd, struct pa
 	if (pc_istrading(sd) || pc_isvending(sd))
 		return;
 
-	int i;
-	uint8 buf[38 + PARTY_BOOKING_JOBS * 2];
+	uint8 buf[38 + MAX_PARTY_BOOKING_JOBS * 2];
 
 	nullpo_retv(sd);
-	if(pb_ad == NULL) return;
+	if (pb_ad == NULL) return;
```

Comment:
```suggestion
	nullpo_retv(pb_ad);
```
And move nullpo checks to top.

Reply:
updated